feature_engine/_docstrings/class_inputs.py

```diff
@@ -13,3 +13,8 @@
         contain missing values. If 'ignore', missing data will be ignored when learning
         parameters or performing the transformation.
         """.rstrip()
+
+_variables_categorical_docstring = """variables: list, default=None
```

Comment:
this doctring already exists within the encoding folder. I would remove this and import the existing docstring.